pyproject.toml

```diff
@@ -101,6 +101,49 @@ homepage = 'https://mne.tools/mne-lsl'
 source = 'https://github.com/mne-tools/mne-lsl'
 tracker = 'https://github.com/mne-tools/mne-lsl/issues'
 
+[tool.cibuildwheel]
+archs = "native"
+before-all = "bash {project}/tools/cibw_before_all.sh"
+build = ["cp310-*"]  # abi3 wheel, can build for a single Python version
+skip = ["*-musllinux*", "pp*"]  # no PyPy support for pp310 for SciPy
+test-command = "python -u {project}/tools/cibw_test_command.py"
+test-requires = ["cffi"]  # we check that the token CFFI binding works
+
+[tool.cibuildwheel.linux]
+repair-wheel-command = """\
+LD_LIBRARY_PATH=/project/liblsl/build \
+auditwheel repair -w {dest_dir} {wheel} \
+"""
+
+[tool.cibuildwheel.linux.environment]
+C_INCLUDE_PATH = "/project/liblsl/include"
+LIBRARY_PATH = "/project/liblsl/build"
+MNE_LSL_BUILD_FFI = "1"
+RUNNER_OS = "Linux"  # could use environment-pass but this facilitates local testing
+
+[tool.cibuildwheel.macos]
+repair-wheel-command = """\
+DYLD_LIBRARY_PATH=$GITHUB_WORKSPACE/liblsl/build \
+delocate-wheel -w {dest_dir} -v {wheel} --require-archs {delocate_archs} \
+"""
+
+[tool.cibuildwheel.macos.environment]
+CPATH = "$GITHUB_WORKSPACE/liblsl/include"
+LIBRARY_PATH = "$GITHUB_WORKSPACE/liblsl/build"
```

Comment:
Those paths + the missing `RUNNER_OS` variable for macOS and Windows should fail the build process on a local macOS or Windows machine, right? Couldn't we use the `cibuildwheel` variable `{project}` as well here?

Reply:
Yes local builds we'd have to do a bit more wrangling. Easiest option would be to tell people to set `GITHUB_WORKSPACE` to their project path. I don't think the `{project}` substitution worked (I think I tried it at some point?) but we could push a commit to double check!